spec-main/window-helpers.ts

```diff
@@ -2,31 +2,37 @@ import { expect } from 'chai'
 import { BrowserWindow, WebContents } from 'electron'
 import { emittedOnce } from './events-helpers';
 
+async function ensureWindowIsClosed(window) {
+  if (window && !window.isDestroyed()) {
+    if (window.webContents && !window.webContents.isDestroyed()) {
+      // If a window isn't destroyed already, and it has non-destroyed WebContents,
+      // then calling destroy() won't immediately destroy it, as it may have
+      // <webview> children which need to be destroyed first. In that case, we
+      // await the 'closed' event which signals the complete shutdown of the
+      // window.
+      const isClosed = emittedOnce(window, 'closed')
+      window.destroy()
```

Comment:
```
window.setClosable(true)
window.close()
```

there is nothing wrong with calling `destroy` here but the api users adopt is `close` for `BrowserWindow`, it would be good to test that.

Reply:
This is not a test, this is cleanup. If we want to test `close()`, we should be explicit and write a test for it, I think.

Reply:
I was thinking, its not possible to test all different scenarios of closing a window, but adding it to a helper class will atleast cover a bunch of cases. Originally `destroy` was exposed to cleanup c++ classes from js land. But considering your point, yeah its better to explicitly test `close`.